api/api.yaml

```diff
@@ -1352,6 +1352,270 @@ paths:
             application/problem+json:
               schema:
                 $ref: "../problem/problem.yaml#/components/schemas/ApiProblem"
+  /projects/{projectIdOrName}/integrations/{integrationId}/objectMetadata:
```

Comment:
@eberle1080 please revise to object-metadata, since our other endpoints don't use camel case in URLs (e.g. provider-app)